.github/workflows/make-branch.yaml

```diff
@@ -8,7 +9,7 @@ on:
         required: true
       branchfrom:
         description: 'The source branch from which to branch, eg., main'
-        default: 'master'
+        default: 'main'
```

Comment:
I see this is running on ubuntu-latest. Should that be 20.04 for consistency with all other build scripts?